postgres/README.md

```diff
@@ -87,26 +87,26 @@ To configure this check for an Agent running on a host:
       #
       - host: localhost
 
-        ## @param port - integer - required
-        ## Port to use when connecting to PostgreSQL.
+        ## @param port - integer - optional - default: 5432
+        ## The port to use when connecting to PostgreSQL.
         #
-        port: 5432
-
-        ## @param user - string - required
-        ## Datadog Username created to connect to PostgreSQL.
+        # port: 5432
+    
+        ## @param username - string - required
+        ## The Datadog username created to connect to PostgreSQL.
         #
         username: datadog
-
-        ## @param pass - string - required
-        ## Password associated with the Datadog user.
+    
+        ## @param password - string - optional
+        ## The password associated with the Datadog user.
         #
-        password: "<PASSWORD>"
-
+        # password: <PASSWORD>
```

Comment:
Is commenting out the password parameter intentional? I ask because you didn't mention it in the PR description.

Reply:
yes, this was in the old examples: https://github.com/DataDog/integrations-core/blob/7.19.x/postgres/datadog_checks/postgres/data/conf.yaml.example. (v.7.19) - now is optional.